deployment/tf/modules/yc/logs/configs/input-kubernetes.conf

```diff
@@ -1,8 +1,8 @@
 [INPUT]
```

Comment:
Here I've disabled stdout scan from containers. Of course, stdout will be still visible in k8s, but not in our logging UI

Reply:
stdout - ok
but we still need stderr, because JVM writes its error to the stderr

Reply:
Okay, I'll create a ticket. It requires some time to figure out, how to do clean stderr logging without _garbage_ logs